packages/fxa-auth-server/test/scripts/prune-pocket-access-tokens.js

```diff
@@ -0,0 +1,312 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+const { assert } = require('chai');
+const util = require('node:util');
+const exec = util.promisify(require('node:child_process').exec);
+const path = require('path');
+const buf = require('buf').hex;
+const hex = require('buf').to.hex;
+const ScopeSet = require('fxa-shared').oauth.scopes;
+const crypto = require('crypto');
+
+const db = require('../../lib/oauth/db');
+
+const POCKET_CLIENT_IDS = [
+  '749818d3f2e7857f', // pocket-web
+  '7377719276ad44ee', // pocket-mobile
+];
+
+function randomString(len) {
+  return crypto.randomBytes(Math.ceil(len)).toString('hex');
+}
+
+describe('#integration - scripts/prune-pocket-access-tokens', function () {
+  this.timeout(15000);
+
+  const cwd = path.resolve(__dirname, '../..');
+
+  let pocketWebClientId;
+  let pocketMobileClientId;
+  let nonPocketClientId;
+  let userId;
+  const scope = ScopeSet.fromArray(['profile']);
+
+  // Keep track for DB cleanup purposes
+  let createdPocketWeb = false;
+  let createdPocketMobile = false;
+
+  before(async function () {
+    pocketWebClientId = buf(POCKET_CLIENT_IDS[0]);
+    pocketMobileClientId = buf(POCKET_CLIENT_IDS[1]);
+    nonPocketClientId = buf(randomString(8));
+    userId = buf(randomString(16));
+
+    // Pocket clients may already exist in CI, so wrap in try/catch.
```

Comment:
Still squinting at this - without these checks all the tests pass locally for me but in CI, I get an `ER_DUP_ENTRY`, so this seemed fine... everything is passing now locally and in CI but now I'm wondering if I'm deleting something in this test DB and then not restoring it at the end heh, I could do a check and add it back at the end 🤔